docs/source/dataframe-parquet.rst

```diff
@@ -103,18 +103,20 @@ disable loading the ``_metadata`` file by specifying
    ...      ignore_metadata_file=True  # don't read the _metadata file
    ... )
 
-If no ``_metadata`` file is present, Dask will load each parquet file
-individually as a partition in the Dask dataframe. This is performant provided
-all files are of reasonable size.
+Partition Size
+~~~~~~~~~~~~~~
+
+By default, Dask will load each parquet file individually as a partition in
+the Dask dataframe. This is performant provided all files are of reasonable size.
 
 We recommend aiming for 10-250 MiB in-memory size per file once loaded into
 pandas. Too large files can lead to excessive memory usage on a single worker,
 while too small files can lead to poor performance as the overhead of Dask
-dominates. If you need to read a parquet dataset composed of many large files
-and lacking a ``_metadata`` file, you can pass ``split_row_groups=True`` to
-have Dask partition your data by *row group* instead of by *file*. Note that
-this can be *extremely* slow in large datasets, as the a footer needs to be
-loaded from every file in the dataset.
+dominates. If you need to read a parquet dataset composed of many large files,
+you can pass ``split_row_groups=True`` to have Dask partition your data by
+*row group* instead of by *file*. Note that this can be *extremely* slow in
+large datasets, as the a footer needs to be loaded from every file in the
```

Comment:
That’s a fair point @bryanwweber - It does feel like an exaggeration to say that `split_row_groups=True` is likely to be "extremely slow" for all large datasets. There is usually only a noticeable difference on remote storage (or when a local dataset is >1000 files or so).  I think the existing wording was to motivate users to explicitly disable `split_row_groups` if/when they are experiencing performance problems. Perhaps we can change this to “Note that this approach will typically not scale as well as `split_row_groups=False`, as the footer…”, since we are changing the `split_row_groups` default anyway?

I don't think we should throw out an arbitrary number to define "large" if we can avoid it.

Reply:
That sounds good @rjzamora, I think reframing as the default value is changing is appropriate here. I also think "doesn't scale well" is a good phrasing.

Reply:
Thanks for the useful feedback Bryan - Feel free to suggest further changes to the updated text.

Reply:
Looks good to me now @rjzamora! Thanks!